### DISCUSSION TITLE
custom colorPalette for a single component

### DISCUSSION THREAD

**Discussion by danchitnis** [chakra-ui/chakra-ui#9185] [Q&A] [answered]
In Chakra V3, for example, if I have CheckBox components, how can I give it a custom `rgb(r,g,b)` instead of the predefined `colorPalette`? This is for that component only, not the entire page

**Comment by segunadebayo** [comment, marked as answer]
The color palette feature requires a range of colors 50-950 to work correctly. You can't pass a custom rgb color.

Instead you should follow this guide
https://www.chakra-ui.com/guides/theming-custom-colors
